<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFirmware</class>
 <widget class="QDialog" name="CFirmware">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firmware Tools</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qthid.qrc">
    <normaloff>:/qthid/images/fw.png</normaloff>:/qthid/images/fw.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Ready...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="toolTip">
          <string>Currently selected firmware.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="selectButton">
       <property name="toolTip">
        <string>Select a firmware file on disk</string>
       </property>
       <property name="text">
        <string>&amp;Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="toolTip">
        <string>Upload the selected firmware to the FCD</string>
       </property>
       <property name="text">
        <string>&amp;Upload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="verifyButton">
       <property name="toolTip">
        <string>Compare the selected firmware with the firmware in the FCD</string>
       </property>
       <property name="text">
        <string>&amp;Verify</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qthid.qrc"/>
 </resources>
 <connections/>
</ui>
